### PR TITLE
Adds date to systemlog entries

### PR DESCRIPTION
**Pull request by cgarwood**
Currently the system log only shows the time an error popped up. This adds the date to the entries in the system log on the dev-info panel as well.

### CODE REVIEW

**Comment by c727** [comment]
you see the date if you click on the item

**Comment by c727** [comment]
...if you want the date there add it only if != current day

**Comment by cgarwood** [comment]
Updated, should only return the date if it's not the current day, otherwise it just returns the time.

Also fixed the margin on the full error log while I was in the file

**Comment by c727** [comment]
- we use spaces instead of tabs
- we use es6/es2015 style so let/const instead of var
you can replace the existing function:
```javascript
formatTime(date) {
  const today = new Date().setHours(0, 0, 0, 0);
  const dateTime = new Date(date * 1000);

  return dateTime.setHours(0, 0, 0, 0) < today ?
    window.hassUtil.formatDateTime(dateTime) : window.hassUtil.formatTime(dateTime);
}
```

File: panels/dev-info/ha-panel-dev-info.html [hunk 1]
```diff
@@ -309,7 +310,16 @@ <h2>Log Details ([[selectedItem.level]])</h2>
...
-    return window.hassUtil.formatTime(new Date(date * 1000));
+    const today = new Date().setHours(0, 0, 0, 0);
+    const dateTime = new Date(date * 1000);
+    const dateTimeDay = new Date(date * 1000).setHours(0, 0, 0, 0);
+
+    return dateTimeDay < today ?
+      window.hassUtil.formatDateTime(dateTime) : window.hassUtil.formatTime(dateTime);
+  }
+
+  formatDateTime(date) {
```
Comment (c727) [inline panels/dev-info/ha-panel-dev-info.html]: not required anymore

File: panels/dev-info/ha-panel-dev-info.html [hunk 2]
```diff
@@ -309,7 +310,16 @@ <h2>Log Details ([[selectedItem.level]])</h2>
   }
 
   formatTime(date) {
-    return window.hassUtil.formatTime(new Date(date * 1000));
+    const today = new Date().setHours(0, 0, 0, 0);
+    const dateTime = new Date(date * 1000);
+    const dateTimeDay = new Date(date * 1000).setHours(0, 0, 0, 0);
```
Comment (c727) [inline panels/dev-info/ha-panel-dev-info.html]: we would use `dateTime.setHours(0, 0, 0, 0);` here but I think my code was fine already as we shouldn't define var's if we only need it once. `today` is more for a better readability

Comment (c727) [reply panels/dev-info/ha-panel-dev-info.html]: ahh correct

**Review by c727 — APPROVED** [review]

**Comment by balloob** [comment]
@cgarwood can you include screenshots when making UI changes?

**Pull request merged**